_posts/2015-04-14-github-101.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "GitHub 101"
+date:   2015-04-14 21:30:00
+author: "Mircea Brățan"
+categories: tutorial
+---
+
+Pentru a veni în întâmpinarea celor care nu sunt familiarizați cu sistemul de versionare Git am pregătit acest tutorial care vă va arăta pașii care trebuie urmați pentru a încărca proiectele voastre pe GitHub.
+
+<!-- more -->
+
+Primul pas este crearea unui cont pe [GitHub](https://github.com/), completând un username, adresa de email și o parolă. La următorul pas va trebui să selectați tipul de cont dorit. Implicit este selectat contul gratuit, care permite doar crearea de repository-uri publice. Apăsați ”Finish sign up”.
+
+Acesta este un moment bun pentru a valida adresa de email aleasă: tot ce trebuie să faceți este să dați click pe link-ul din interiorul email-ului primit de la GitHub.
+
+Pagina de start a GitHub va conține acum rubrica GitHub Bootcamp, o colecție de resurse suplimentare despre utilizarea GitHub. Noi ne vom axa pe crearea unui repository. Pentru aceasta dați click pe *+ New Repository*
+
+{: .center}
+![GitHub homepage](/assets/images/github-101/01-dashboard.png)
+
+
+
+Următorul pas este alegerea unui nume pentru repository. În câmpul „Description” puteți adăuga o scurtă descriere a proiectului. Nu uitați să bifați „Initialize this repository with a README”. Optional puteți alege, în partea de jos a paginii, adăugarea unui fișier .gitignore și/sau a unei licențe.
+
+Fișierul .gitignore este folosit pentru ca git să ignore fișierele pe care nu le doriți în repository, de exemplu: fișiere generate la compilare, fișiere private, etc. Mai multe detalii puteți găsi aici: [help.github.com](https://help.github.com/articles/ignoring-files/).
```

Comment:
Poate suna mai bine `la ignorarea fisierelor, care nu le doriti in repository, de catre git. De exemplu: `